.cloudbuild/graal-build.yaml

```diff
@@ -4,16 +4,19 @@ steps:
     entrypoint: bash
     args: ['./.cloudbuild/graal-build-script.sh']
 
+
   # Start emulators
   - name: 'gcr.io/cloud-builders/docker'
     args: ['run', '-d', '--network=cloudbuild', '--name=spanner-emulator', '-p', '9010:9010', '-p', '9020:9020', 'gcr.io/cloud-spanner-emulator/emulator:1.1.1']
   - name: 'gcr.io/cloud-builders/docker'
     args: ['run', '-d', '--network=cloudbuild', '--name=firestore-emulator', 'google/cloud-sdk', 'sh', '-c', 'gcloud beta emulators firestore start --host-port=0.0.0.0:9010']
   - name: 'gcr.io/cloud-builders/docker'
     args: ['run', '-d', '--network=cloudbuild', '--name=bigtable-emulator', 'google/cloud-sdk', 'sh', '-c', 'gcloud beta emulators bigtable start --host-port=0.0.0.0:9010']
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['run', '-d', '--network=cloudbuild', '--name=datastore-emulator', 'google/cloud-sdk', 'sh', '-c', 'gcloud beta emulators datastore start --host-port=0.0.0.0:9010']
   # Wait for emulators to start
   - name: jwilder/dockerize:0.6.1
-    args: ['dockerize', '-timeout=40s', '-wait=tcp://spanner-emulator:9010', '-wait=tcp://firestore-emulator:9010', '-wait=tcp://bigtable-emulator:9010']
+    args: ['dockerize', '-timeout=40s', '-wait=tcp://spanner-emulator:9010', '-wait=tcp://firestore-emulator:9010', '-wait=tcp://bigtable-emulator:9010', '-wait=tcp://datastore-emulator:9010']
```

Comment:
You could probably speed up the builds below by adding the `waitFor` attribute to each of these. IIUC, `waitFor: -` will start this step immediately (and since these emulators don't need the first step, they don't need to wait for it to complete, right?)

Reply:
Thanks for the tip -- Done.